Common/src/CConfig.cpp

```diff
@@ -3302,6 +3312,47 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     VolumeOutputFiles[2] = OUTPUT_TYPE::SURFACE_PARAVIEW_XML;
   }
 
+  /*--- Set the default output frequencies ---*/
+  if (!OptionIsSet("OUTPUT_WRT_FREQ")){
+    nVolumeOutputFrequencies = nVolumeOutputFiles;
+    VolumeOutputFrequencies = new unsigned long [nVolumeOutputFrequencies];
+
+    /*---  Using default frequency of 250 for all files when steady, and 1 for unsteady. ---*/
+    for (auto iVolumeFreq = 0; iVolumeFreq < nVolumeOutputFrequencies; iVolumeFreq++){
+      if (Time_Domain) 
+        VolumeOutputFrequencies[iVolumeFreq] = 1; 
+      else
+        VolumeOutputFrequencies[iVolumeFreq] = 250; 
+    }
+  } else {
+    /*--- Check how many frequencies are in the list. 
+     *    If 1 then use this frequency for all output files. 
+     *    If 2 then use the first value for the first output file, the second for the other output files.
+     *    This is useful to define 1 frequency for the restart file and 1 frequency for all the visualization files.  ---*/
+    if ((nVolumeOutputFrequencies <= 2) && (nVolumeOutputFrequencies != nVolumeOutputFiles)) {
+
+      const auto firstOutputWriteFreq = VolumeOutputFrequencies[0];
+      auto secondOutputWriteFreq = VolumeOutputFrequencies[0];
+
+      if (nVolumeOutputFrequencies == 2) {
+        secondOutputWriteFreq = VolumeOutputFrequencies[1];
+      } 
+
+      /*--- we recreate the OutputFrequencies ---*/
+      nVolumeOutputFrequencies = nVolumeOutputFiles;
+      delete [] VolumeOutputFrequencies;
+      VolumeOutputFrequencies = new unsigned long [nVolumeOutputFrequencies];
+
+      VolumeOutputFrequencies[0] = firstOutputWriteFreq; 
+      
+      for (auto iVolumeFreq = 1; iVolumeFreq < nVolumeOutputFrequencies; iVolumeFreq++){
+        VolumeOutputFrequencies[iVolumeFreq] = secondOutputWriteFreq; 
+      }
+    } else if (nVolumeOutputFrequencies != nVolumeOutputFiles) {
+      SU2_MPI::Error("Number of entries in OUTPUT_WRT_FREQ is not equal to number of entries in OUTPUT_FILES.", CURRENT_FUNCTION);
+    }
```

Comment:
I think this can be completely error-free for the user.
```suggestion
    auto* newFrequencies = new unsigned long[nVolumeOutputFiles];
    for (unsigned short i = 0; i < nVolumeOutputFrequencies; ++i) {
      newFrequencies[i] = VolumeOutputFrequencies[i];
    }
    for (auto i = nVolumeOutputFrequencies; i < nVolumeOutputFiles; ++i) {
      newFrequencies[i] = newFrequencies[i-1];
    }
    delete [] VolumeOutputFrequencies;
    VolumeOutputFrequencies = newFrequencies;
    nVolumeOutputFrequencies = nVolumeOutputFiles;
```